emscripten.py

```diff
@@ -1728,7 +1728,8 @@ def create_receiving(function_table_data, function_tables_defs, exported_impleme
   if not shared.Settings.DECLARE_ASM_MODULE_EXPORTS:
     receiving += 'exportAsmFunctions(asm);'
   else:
-    if not shared.Settings.SWAPPABLE_ASM_MODULE:
+    swappable = (shared.Settings.WASM and shared.Settings.WASM_ASYNC_COMPILATION) and not shared.Settings.MINIMAL_RUNTIME
```

Comment:
what does swappable mean here, and below? a comment could help, as before the reader could refer to the meaning of the setting in settings.js.

Reply:
Done